pytubefix/contrib/search.py

```diff
@@ -315,7 +317,6 @@ def fetch_and_parse(self, continuation=None):
                                 'reelWatchEndpoint']['videoId']
 
                         shorts.append(YouTube(f"https://www.youtube.com/watch?v={video_id}",
-                                              client=self.client,
```

Comment:
Removing the `self.client` parameter from the constructor will make it impossible to pass the client.